src/js/hca-rjsf/helpers.jsx

```diff
@@ -0,0 +1,10 @@
+import { transformForSubmit } from '../common/schemaform/helpers';
+
+export function transform(formConfig, form) {
+  const formData = transformForSubmit(formConfig, form);
+  return JSON.stringify({
+    educationBenefitsClaim: {
```

Comment:
Mmm....not sure what this should be, but I'm guessing probably not `educationBenefitsClaim`.